src/odb/include/odb/geom_boost.h

```diff
@@ -280,48 +280,42 @@ struct tag<odb::Polygon>
 template <>
 struct ring_mutable_type<odb::Polygon>
 {
-  using type = std::vector<odb::Point>;
+  using type = std::vector<odb::Point>&;
 };
 
 template <>
 struct ring_const_type<odb::Polygon>
 {
-  using type = const std::vector<odb::Point>;
+  using type = const std::vector<odb::Point>&;
 };
 
 template <>
 struct interior_const_type<odb::Polygon>
 {
-  using type = const std::vector<std::vector<odb::Point>>;
+  using type = const std::vector<std::vector<odb::Point>>&;
 };
 
 template <>
 struct interior_mutable_type<odb::Polygon>
 {
-  using type = std::vector<std::vector<odb::Point>>;
+  using type = std::vector<std::vector<odb::Point>>&;
 };
 
 template <>
 struct exterior_ring<odb::Polygon>
 {
-  static std::vector<odb::Point> get(odb::Polygon& p) { return p.getPoints(); }
-  static const std::vector<odb::Point> get(const odb::Polygon& p)
+  static auto& get(odb::Polygon& p) { return p.getPointsMutable(); }
```

Comment:
what is making us have to support this? Since it wasn't needed before

Reply:
`boost::geometry::difference(bounding_rect, block->_die_area, results);` Specifically has a template that says these methods must be references and const references in order to support swap operations. 

Returning these by value I guess is technically incorrect https://stackoverflow.com/questions/67204323/how-to-adapt-a-custom-polygon-type-in-boost-geometry